### ISSUE TITLE
Large files

### ISSUE THREAD

**Issue by e-s-a-i** [lolomolo/LongboardMarkII#1]
GitHub is not a place to host videos, doing so makes contributing difficult. I wouldn't like to download 500MB of content just to propose a small change.

**Comment by Daandreev** [comment]
Makes sense. They are important for the project but I can leave them on my
laptop
